engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/repositories/SuspendedExecutionsRepository.java

```diff
@@ -34,4 +34,9 @@
 
     @Query("from SuspendedExecution se where se.numberOfBranches=size(se.finishedBranches)")
     public List<SuspendedExecution> findFinishedSuspendedExecutions(Pageable pageRequest);
+
+    @Query("select  se from SuspendedExecution se " +
+            "left join io.cloudslang.orchestrator.entities.ExecutionState es " +
+            "on se.executionId=es.executionId")
```

Comment:
= sign must be preceded and followed by a space to have really nice indentation :)